tests/modeladmin/test_checks.py

```diff
@@ -1309,3 +1309,29 @@ class BandAdmin(ModelAdmin):
             'custom_permission_action action.',
             id='admin.E129',
         )
+
+    @classmethod
```

Comment:
Why is this a classmethod?

Reply:
Would it be okay if I made it a staticmethod? (To limit its scope and share code between the two tests.)

Reply:
Well, I wrote this check with a specific code in mind (cf. https://code.djangoproject.com/ticket/29711) and I wanted it to showcase clearly in what way such an unintended behaviour can occur and confuse a developer (``actions = (action, action)`` resulting in one action is kind of obvious).

Of course it makes sense only when someone reads the source-code of the checks and I have no idea if that's important enough to keep it as it is.